app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/MappingWizardDatastoresStep.js

```diff
@@ -151,7 +151,11 @@ MappingWizardDatastoresStep.propTypes = {
 };
 MappingWizardDatastoresStep.defaultProps = {
   clusterMappings: [],
-  fetchStoragesUrls: {},
+  fetchStoragesUrls: {
```

Comment:
I figured it was used for configuration of the plugin. So if there is scenario, when it might be required to change those values (like using it in different product) i would maybe recommend to move them to redux and have some "read only" reducer for configuration.  I think all components that are using this are connected anyway, so basically only the origin of the data would change. We would not have move the data to `defaultProps` and we could eventually delete the whole `coreComponents` file.

Only other thing that is depending on this are the routes i think, but it should not be hard to replace.

Reply:
@priley86 ok i will move it to default props for now. You guys can decide which path you want to take later. Also ,removing the original component registry should not be part of this PR. That is maybe something for future refactoring.

Reply:
Works!